### ISSUE TITLE
JavaScript reserved words being generated as variable names

### ISSUE THREAD

**Issue by aacitelli** [astahmer/openapi-zod-client#125]
Example:
![image](https://user-images.githubusercontent.com/30580900/233504376-29a27e17-ad21-4148-b551-cada8f1b5b72.png)

Generated from the official GitHub REST API OpenAPI spec: https://github.com/github/rest-api-description/blob/main/descriptions/api.github.com/api.github.com.yaml

**Comment by astahmer** [comment]
good catch ! feel free to open a PR, the relevant code should be around [here](https://github.com/astahmer/openapi-zod-client/blob/20e959358b8b270465e9648b26d5ceaeb33f1c78/lib/src/getZodiosEndpointDefinitionList.ts#L69)
